File: docs/competitors/pascal/SumNumbers/SumNumbers.pas
program SumNumbers;
var
  s: string;
  x, sum: real;
begin
  sum := 0;
  writeln('Enter real numbers (empty line to finish):');
  readln(s);
  while s <> '' do
  begin
    x := real.Parse(s);
    sum := sum + x;
    readln(s);
  end;
  writeln('Sum = ', sum);
end.

File: docs/competitors/c++/SumNumbers/SumNumbers.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string s;
    double x, sum = 0;

    cout << "Enter real numbers (empty line to finish):" << endl;
    getline(cin, s);

    while (!s.empty()) {
        x = stod(s);
        sum += x;
        getline(cin, s);
    }

    cout << "Sum = " << sum << endl;
    return 0;
}